src/event-handlers.js

```diff
@@ -399,6 +399,17 @@ export default {
     this.setState({ useAlternateEndDayButtonPosition })
   },
 
+  handleHideHomeScreenChange(
+      _e,
+      hideHomeScreen
+  ) {
+    if (this.state.stageFocus === 'HOME') {
+      this.focusNextView()
+    }
+
+    this.setState({ hideHomeScreen })
+  },
```

Comment:
Though this logic is fairly simple, I think it's worth adding some unit testing for this event handler in case it ever becomes more complex.

Reply:
I will get these done tomorrow as I haven't written Javascript unit tests before so I need to go inhale some knowledge before I stab wildly into the dark. 😅

Reply:
Sounds good! Let us know if you have any questions. IMO the project's current testing situation is decent, if a little hit-or-miss. There are some odd aspects of how a few things are set up related to mocks, but I don't think you'll run into much (if any) of that in this PR.

(FWIW, I take full responsibility for the more subpar aspects of Farmhand's testing suite. @lstebner has improved it considerably overall since he joined the project! 🙂)